app/layout.tsx

```diff
@@ -17,9 +18,17 @@ const sora = Sora({
 });
 
 export default function RootLayout({ children }: { children: ReactNode }) {
+  const headersList = headers();
+
+  const headerUrl = headersList.get('x-url') || '';
```

Comment:
why did you choose to use `headers` instead of `usePath
- learn more: https://nextjs.org/docs/app/api-reference/functions/use-pathname

<img width="688" alt="Screenshot 2024-06-28 at 10 42 00 AM" src="https://github.com/Coop-Records/sonata/assets/23249402/0a50b269-e05c-49e8-af7e-6e4ec627fbe6">